python/cog/server/worker.py

```diff
@@ -448,6 +453,9 @@ async def _apredict(
                     self._events.send(PredictionOutputType(multi=False))
                     self._events.send(PredictionOutput(payload=make_encodeable(output)))
 
+    def _loop_scope(self) -> Scope:
```

Comment:
Not a big deal but why is this called loop_scope? Isn't it a prediction scope?

Reply:
It's a scope to be used inside the loop. At least that was my reasoning :)